rpc/backend/tx_info.go

```diff
@@ -174,7 +174,7 @@ func (b *Backend) GetTransactionReceipt(hash common.Hash) (map[string]interface{
 	res, additional, err := b.GetTxByEthHash(hash)
 	if err != nil {
 		b.Logger.Debug("tx not found", "hash", hexTx, "error", err.Error())
-		return nil, err
```

Comment:
Is it the only type of error you can have?
Wondering if there could be some filtering we could do here